ner_v2/detectors/numeral/number_range/standard_number_range_detector.py

```diff
@@ -126,7 +128,7 @@ def _tag_number_in_text(self, processed_text):
             i want to buy __number__1 apples and more than __number__0 bananas
         """
         tagged_number_text = processed_text
-        sorted_number_detected_map = sorted(self.number_detected_map.items(), key=lambda kv: len(kv[1].original_text),
+        sorted_number_detected_map = sorted(list(self.number_detected_map.items()), key=lambda kv: len(kv[1].original_text),
```

Comment:
E501 line too long (124 > 119 characters)

Reply:
Fix this lint error